package.json

```diff
@@ -63,9 +63,7 @@
     ],
     "files": [
         "src/",
-        "!src/test/",
-        "dist/",
-        "!dist/test/"
```

Comment:
Why? we don't want to ship the tests in the NPM bundle...

Reply:
I had moved the test outside of src, so we can safely ship production and source code in NPM module

Reply:
Ok silly me